server/models/migrations/v0.8.0-v0.9.0/migrate.sql

```diff
@@ -964,3 +964,35 @@ WHERE id = 199;
 UPDATE role_unit SET unit_id = 199 WHERE unit_id = 159;
 DELETE FROM unit WHERE id = 159;
 DELETE FROM report WHERE id = 9;
+
+
+-- author: @mbayopanda
+-- stock assignment feature
+-- the stock assign table
+DROP TABLE IF EXISTS `stock_assign`;
+CREATE TABLE `stock_assign` (
+  `uuid`              BINARY(16) NOT NULL,
+  `lot_uuid`          BINARY(16) NOT NULL,
+  `entity_uuid`       BINARY(16) NOT NULL,
+  `depot_uuid`        BINARY(16) NOT NULL,
```

Comment:
Is the `depot_uuid` where the stock has come from when it is assigned?

Reply:
Yes